.github/workflows/bot.yml

```diff
@@ -15,6 +15,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
+        java: [8, 8.0.192]
```

Comment:
do we still need this? would just `java: '8'` work?

Reply:
think it should go to the `include:` list below along with "scala" and "spark"? just `'8'`?